src/main/webapp/business.html

```diff
@@ -1,3 +1,32 @@
+<!DOCTYPE html>
 <html>
-Business page
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <title>Business Page</title>
+        <script src="script.js"></script>
+    </head>
+    <body onload="initMap()">
+        <div> 
+            Black-Owned Businesses in Minneapolis 
+            <ul>
+                <li>Afro Deli- 720 Washington Ave SE</li>
```

Comment:
Longer term, you should think about how this can be stored in the backend and fetched here from the backedn. It could be just a file stored in the backend. That would be easier to maintain, especially if you wan tto alpaphabetially order the list etc.With that it mind, in a follow up PR maybe populate this from javascript.

Reply:
That makes sense, similar to what we did for the comments on our portfolio.